app/clients/freshdesk.py

```diff
@@ -51,20 +53,19 @@ def _generate_zd_ticket(self) -> Dict[str, Union[str, int, List[str]]]:
             'tags': self.contact.tags
         }
 
-    def __init__(self, contact: ContactRequest):
-        self.contact = contact
-
     def send_ticket(self) -> int:
         try:
             api_url = current_app.config['FRESH_DESK_API_URL']
             if not api_url:
                 raise NotImplementedError
 
-            ticket = self._generate_zd_ticket()
+            # The API and field definitions are defined here:
+            # https://developer.zendesk.com/rest_api/docs/support/tickets
             response = requests.post(
-                f"{api_url}/api/v2/tickets",
-                json=ticket,
-                auth=HTTPBasicAuth(current_app.config['FRESH_DESK_API_KEY'], "x")
+                urljoin(api_url, '/api/v2/tickets'),
+                json=self._generate_ticket(),
+                auth=HTTPBasicAuth(current_app.config['FRESH_DESK_API_KEY'], "x"),
+                timeout=30.0
```

Comment:
30s seems a long time, WDYT think about 5?

Reply:
I think this is an arbitrary number, I'm not sure 5 is better than 30. I'll however make the change.